src/components/ProjectList.tsx

```diff
@@ -101,16 +106,14 @@ const ProjectList: React.FC = () => {
   const tags: string[] = Array.from(new Set(projects.flatMap((project) => project.tags)));
   const langs: string[] = Array.from(new Set(projects.flatMap((project) => project.languages)));
 
- 
-
   const filteredProjects: Project[] = projects.filter((project) => {
     const searchMatch: boolean =
       project.name.toLowerCase().includes(search.toLowerCase()) ||
       project.description.toLowerCase().includes(search.toLowerCase());
   
     const tagsMatch: boolean =
       selectedTags.length === 0 ||
-      selectedTags.every((tag) => project.tags.includes(tag));
+      selectedTags.some((tag) => project.tags.includes(tag));
       
     const langsMatch: boolean =
       selectedLangs.length === 0 ||
```

Comment:
Is `langsMatch` intentionally using `selectedLangs.every` instead of `selectedLangs.some`?

Reply:
No, that was still from the previous filtering function - updated now.